openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCard/ActivityFeedCard.tsx

```diff
@@ -168,9 +168,10 @@ const ActivityFeedCard: FC<ActivityFeedCardProp> = ({
           {
             'tw-bg-gray-100': visible,
           }
-        )}>
+        )}
+        ref={containerRef}>
         <Popover
-          align={{ targetOffset: [0, -35] }}
+          align={{ targetOffset: [0, -14] }}
```

Comment:
@chirag-madlani , Is this change required?

Reply:
Yes, Since we change the container position will get effect